### DISCUSSION TITLE
How do I get Agile rates onto dashboard

### DISCUSSION THREAD

**Discussion by SteveEggo-SpecialAgent** [BottlecapDave/HomeAssistant-OctopusEnergy#454] [Q&A] [answered]
I an new to the Home Automation game. I have it up and running on a RaspberryPi, I have installed HomeAssistant-OctopusEnergy and can display the current and previous consumption  figures. Are there any instructions to enable me to display the Agile rates ? I have installed the Add-on File Editor and can see files in the config/custom_components/octopus_energy. I am unsure which files to edit and what to add to them.

**Comment by anothermort** [comment, marked as answer]
Is this what you are after?

https://github.com/lozzd/octopus-energy-rates-card

**Reply by SteveEggo-SpecialAgent** [reply]
Thank you anothermort, I now have Agile Rates on my dashboard. The half hour slots for today all show the same price. I suspect this will fix itself at 4PM when Octopus publish rates for the next day. If I wish to tweak the layout do I just edit the Y:\.storage\lovelace file ? Following an edit of this file do I have to restart Home Automation, or is there a quicker way?

**Reply by SteveEggo-SpecialAgent** [reply]
I have drive Y: mapped to home assistant\config

**Reply by anothermort** [reply]
The only changes I have done are in the card config on the dash board i.e.
![2023-10-10 10_24_30-Victron – Home Assistant — Mozilla Firefox](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/13786570/4b5155fd-a8d6-4c3d-b41f-36dfd46a8e29)
More details on the previous link under - Here's a breakdown of all the available configuration items:

I am pretty new to HA as well so not got that deep into it yet.
Might be worth asking the question on [https://github.com/lozzd/octopus-energy-rates-card](url) if its deeper than config items.
I always restart HA but as I said new to HA so could be a quicker way.

Good luck
